core/src/nucleus/ribosome/api/mod.rs

```diff
@@ -41,8 +41,11 @@ use wasmi::{
 #[repr(usize)]
 #[derive(FromPrimitive, Debug, PartialEq)]
 pub enum ZomeApiFunction {
+    /// Abort
+    Abort = 0,
```

Comment:
Any reason for putting it in first place?
I would like to keep `MissingNo` on top if possible.

Reply:
No prob, will do. I thought maybe it should go there, since it's not really "our thing" ... it's there for the sake of assemblyscript. But it doesn't matter